CONTRIBUTING-development.md

```diff
@@ -0,0 +1,134 @@
+# Coding style
+
+* We almost always use **CamelCase**, when naming functions, variables etc., **not snake_case**.
```

Comment:
Is it worth mentioning variable/struct naming ?
It is not enforced but often it's similar to:
```c
AlterTableCmd *alterTableCommand = NULL;
```
and also about function to variable name:
```c
bool
IsAlterTableRenameStmt(RenameStmt *renameStmt)
{
	bool isAlterTableRenameStmt = false;
```

I'm pretty sure I read something on this topic somewhere on citus (commit/merge comments, ... )

Reply:
Makes sense, I'd be more than happier if you can suggest a change via "Add a suggestion" button so that the doc states this in a way that it makes most sense to community :)

Reply:
I will find the button.

Reply:
Apparently "Add a suggestion" is possible by tagging codeblock with "suggestion", let's try...


```suggestion
* We almost always use **CamelCase**, when naming functions, variables etc., **not snake_case**.

We also have the habits of using a **lowerCamelCase** for some variables named from their type or from their function name, as shown in the examples:

```c
AlterTableCmd *alterTableCommand = NULL;

[...]

bool
IsAlterTableRenameStmt(RenameStmt *renameStmt)
{
	bool isAlterTableRenameStmt = false;
```

```

Reply:
fixed the "suggestion" and applying it now

Reply:
Thanks btw!